chringo/pubspec.lock

```diff
@@ -1,161 +0,0 @@
-# Generated by pub
```

Comment:
My gut tells me that we want the pubspec.lock. Why was it removed? It details exactly which dependencies we have

Reply:
U rite. I thought it was generated but according to [this](https://dart.dev/guides/libraries/private-files), it should be added for application packages.